webrtc-stats.html

```diff
@@ -1066,6 +1078,27 @@
                   section 6.4.1.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>lastPacketSentTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the timestamp at which the last packet was sent for this media stream track.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>averageRTCPInterval</code></dfn> of type <span class=
```

Comment:
average RTCP is calculated by the endpoint for scheduling RTCP report. So this is about exposing an existing value from the RTP stack. RFC3550/RFC4585 has:
+ target RTCP interval
+ average RTCP interval